File: app/src/main/java/vn/loitp/app/activity/function/viewdraghelpersimple/ViewDragHelperSimpleActivity.java
package vn.loitp.app.activity.function.viewdraghelpersimple;

import android.os.Bundle;

import com.core.base.BaseFontActivity;

import loitp.basemaster.R;

public class ViewDragHelperSimpleActivity extends BaseFontActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    protected boolean setFullScreen() {
        return false;
    }

    @Override
    protected String setTag() {
        return getClass().getSimpleName();
    }

    @Override
    protected int setLayoutResourceId() {
        return R.layout.activity_view_draghelper_simple;
    }
}

File: app/src/main/java/vn/loitp/app/activity/function/viewdraghelpersimple/ViewDragHelperSimpleActivity.kt
package vn.loitp.app.activity.function.viewdraghelpersimple

import com.loitpcore.annotation.IsFullScreen
import com.loitpcore.annotation.LogTag
import com.loitpcore.core.base.BaseFontActivity
import vn.loitp.app.R

@LogTag("ViewDragHelperSimpleActivity")
@IsFullScreen(false)
class ViewDragHelperSimpleActivity : BaseFontActivity() {

    override fun setLayoutResourceId(): Int {
        return R.layout.activity_func_view_draghelper_simple
    }
}
